src/inverted-index.js

```diff
@@ -0,0 +1,160 @@
+import async from 'async';
+import contentFilter from './content-filter';
+import makeAnIndex from './make-index';
+
+const makeIndex = makeAnIndex.makeIndex;
+
+/**
+ * Implementing the Inverted Index search procedure
+ */
+class InvertedIndex {
+  /**
+   * This creates an instance of inverted index, and initializes the index
+   */
+  constructor() {
+    this.index = {};
+  }
+  /**
+   * @return {Object} index of supplied document
+   * @param {string} fileName The name of the file whose index is to be created
+   * @param {string} fileContent The content of the file
+   * @param {function} done A callback, whose argument is the returned index
+   */
+  createIndex(fileName, fileContent) {
+    let indexed = false; // Keep track of indexing operation
```

Comment:
I like the fact that you broke down the project into simpler, smaller and manageable portions. Also your use of JS Doc is awesome, I think you might also want to make your JS Doc self explanatory so that inline comment will not be found in your work. It follows best practices